test/net/sourceforge/kolmafia/preferences/PreferencesTest.java

```diff
@@ -559,6 +568,61 @@ public void exerciseIsPerUserGlobalProperty() {
     assertTrue(Preferences.isPerUserGlobalProperty("getBreakfast.PreferencesTestFakeUser"));
   }
 
+  public class timeinThread extends Thread {
+    public timeinThread(String s) {
+      super(s);
+    }
+
+    public void run() {
+      LoginManager.timein(CONCURRENT_USER_NAME);
+    }
+  }
+
+  public class incrementThread extends Thread {
+    public incrementThread(String s) {
+      super(s);
+    }
+
+    public void run() {
+
+      Preferences.increment("counter", 1);
+    }
+  }
+
+  @Disabled
+  @Test
+  public void timeinDoesNotCauseRaceCondition() {
+    String unrelatedPref = "coalmine";
```

Comment:
I was concerned that we might power through the SetPreferences and get the increment to write and kill a different preference.  This was based on anecdotal discussions from users about what prefs were still there after being nuked.  Since I'm using increment, it depends on the current value and is easier to see failures with, but if we were writing a string, we might write only the string in the otherwise blank prefs file...

Should we two assertions, when this is not a failing test.